love/src/components/ESS/AuxTel/AuxTelESS.container.jsx

```diff
@@ -99,30 +100,67 @@ const getGroupNames = (salindexList, option) => {
   return [...new Set(subscriptions)];
 };
 
+const prevParseToArraySensors = (prevParse, option) => {
+  const list = Object.values(prevParse[option] ?? {}) ?? [];
+  const objs = Object.values(list).map((values) => {
+    return Object.values(values);
+  });
+
+  const result = Array.prototype.concat.apply([], objs);
+  const sorted = result.sort((a, b) => {
+    return a.sensorName > b.sensorName ? 1 : (
+      a.sensorName < b.sensorName ? -1 : (
+        a.sensorName === b.sensorName && a.indexArr > b.indexArr ? 1 : (
+          a.indexArr < b.indexArr ? -1 : 0
+        )
+      )
+    );
+  });
+  return sorted;
+}
+
+let prevParse = {};
 const parse = (streams, option) => {
```

Comment:
Also found dificult to understand, can you add documentation and maybe rename the function?